src/runner_manager.py

```diff
@@ -462,6 +464,8 @@ def reconcile(self, quantity: int, resources: VirtualMachineResources) -> int:
             ]
 
             for runner in unhealthy_runners:
+                if self.config.charm_state.is_metrics_logging_available:
```

Comment:
Could be interesting to have an interface method in `config` that abstract how this information is accessed ?